packages/shared/src/components/squads/SquadPageHeader.tsx

```diff
@@ -27,8 +31,19 @@ export function SquadPageHeader({
 }: SquadPageHeaderProps): ReactElement {
   const { sidebarRendered } = useSidebarRendered();
 
+  const sharePostTutorial = useTutorial({
+    key: TutorialKey.SHARE_SQUAD_POST,
+  });
+
+  const isDesktop = useMedia([laptop.replace('@media ', '')], [true], false);
+
   return (
-    <FlexCol className="relative items-center laptop:items-start px-6 pb-20 tablet:pb-20 laptop:pb-14 mb-6 w-full tablet:border-b laptop:px-[4.5rem] min-h-20 border-theme-divider-tertiary">
+    <FlexCol
+      className={classNames(
+        'relative items-center laptop:items-start px-6 pb-20 tablet:pb-20 laptop:pb-14 mb-6 w-full tablet:border-b laptop:px-[4.5rem] min-h-20 border-theme-divider-tertiary',
+        sharePostTutorial.isActive && 'laptop:mb-28 mb-28',
```

Comment:
I think the idea is that this always has the margin now, regardless of wether the post is shown.

Reply:
Checked and we decided to keep as implemented